File: src/main/java-graphql/io/github/graphql/model/RepositoryAffiliationTO.java
package io.github.graphql.model;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:47+0800"
)
public enum RepositoryAffiliationTO {

    COLLABORATOR("COLLABORATOR"),
    ORGANIZATION_MEMBER("ORGANIZATION_MEMBER"),
    OWNER("OWNER");

    private final String graphqlName;

    private RepositoryAffiliationTO(String graphqlName) {
        this.graphqlName = graphqlName;
    }

    @Override
    public String toString() {
        return this.graphqlName;
    }

}

File: src/main/scala-graphql/io/github/graphql/model/RepositoryAffiliationTO.scala
package io.github.graphql.model

import com.fasterxml.jackson.core.`type`.TypeReference

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:00+0800"
)
object RepositoryAffiliationTO extends Enumeration {

    type RepositoryAffiliationTO = Value

    val COLLABORATOR: Value = Value("COLLABORATOR")
    val ORGANIZATION_MEMBER: Value = Value("ORGANIZATION_MEMBER")
    val OWNER: Value = Value("OWNER")

}

class RepositoryAffiliationTOTypeRefer extends TypeReference[RepositoryAffiliationTO.type]
